src/current/_includes/molt/molt-oracle-limitations.md

```diff
@@ -0,0 +1,22 @@
+## Limitations
+
+- Migrations must be performed from a single Oracle schema. You **must** include `--schema-filter` so that MOLT Fetch only loads data from the specified schema. Refer to [Schema and table filtering](#schema-and-table-filtering).
+  - Specifying `--table-filter` is also strongly recommended to ensure that only necessary tables are migrated from the Oracle schema. A userscript is required to use `--table-filter` with an Oracle source.
+
+{% if page.name != "bulk-load.md" %}
+- Replication will not work for tables or column names exceeding 30 characters. This is a [limitation of Oracle LogMiner](https://docs.oracle.com/en/database/oracle/oracle-database/21/sutil/oracle-logminer-utility.html#GUID-7594F0D7-0ACD-46E6-BD61-2751136ECDB4).
+
+- Running DDL on the source or target while replication is in progress can cause replication failures.
+
+- Oracle LogMiner does not support the following data types:
+  - User-defined types (UDTs)
+  - Nested tables
+  - `VARRAY`
+  - `LONGBLOB`/`CLOB` columns (over 4000 characters)
```

Comment:
I thought it does support it but Replicator just doesn't right now.

Reply:
@noelcrl ?